Typos, language, spelling (#55)
edgelesssys/marblerun.sh: content/docs/workflows/define-manifest.md

* `KeyUsage` & `ExtKeyUsage`
* `Subject` (+ children)

> The following X.509 properties can not be specified because they are set by the Coordinator when creating a certificate.

Reviewer: ```suggestion
The following X.509 properties can't be specified because they are set by the Coordinator when creating a certificate.
```
https://docs.microsoft.com/en-us/style-guide/top-10-tips-style-voice#project-friendliness
Since we seem to use `cannot` instead for `can't currently (see line 157), I would suggest leaving it at `cannot` ot to change both.
See https://docs.microsoft.com/en-us/style-guide/word-choice/use-contractions
